### DISCUSSION TITLE
PK Simulation of Bortezomib - half-life adjustment

### DISCUSSION THREAD

**Discussion by JohannaZH** [Open-Systems-Pharmacology/Forum#1597] [Questions & Problems]
Dear all, 
I am on creating of simulation of Bortezomib in PK-Sim and faced few problems which I would like to discuss with you.

1. With which parameter in PK-Sim could I adjust half-life of the compond?

2. I created two simulations, one for dose 1 mg/m2 and one for dose 1.3 mg/m2. To match Cmax in plasma (from literature) which are belonging to these different doses two different LogPs are needed. How is that possible? 

Thank you for your opinions and help. 
Johanna

**Comment by wilbertdew** [comment]
Hi @JohannaZH,

You can use different parameters, for example the lipophilicity as you mentioned. You could also try different calculation methods for the partition coefficients and the cellular permeability. Apart from that, hepatic clearance and renal clearance would be the most logical options to start with, I would think, although I am not aware of the specifics for Bortezomib.
For you second question, first have a look at the non-linearity of you Cmax, is it indeed not proportional to the dose? If not, you could think about saturable clearance, saturable metabolism in the gut or saturable absorption mechanisms.
Hope this helps, you still have lots of options to explore it seems!
Wilbert
